### DISCUSSION TITLE
Multiple bots in one laravel project

### DISCUSSION THREAD

**Discussion by Jorge-S13** [nutgram/nutgram#486] [Q&A]
Hi, can I run multiple bots in the same laravel project? I tried to create two different controllers and routes for them. But when I call the bots, they work one by one. Is it possible to implement such logic? This question has already been asked, but it hasn't been answered.

**Comment by Lukasss93** [comment]
There's already an open issue inside the laravel repository: https://github.com/nutgram/laravel/issues/2
Stay tuned for future updates.

**Reply by Jorge-S13** [reply]
is there a solution at the moment? Can I create multiple bots at the same time? because when i do that my bots don't work at the same time, they work in turn

**Comment by Jorge-S13** [comment]
if I create a new controller and do $bot = new Nutgram(' token '); and describe the logic of bot-1 in this controller, and then create another controller, do $bot = new Nutgram(' token2 '); and describe bot-2 logic in this controller. I will make a route for each of them. So in theory it should work? but it doesn't work.I don't need to create a bot with a console command. I just want to run multiple bots at the same time

**Reply by sergix44** [reply]
Yes, thats perfectly fine, for now the thing that is not supported yet is defining different configurations for different bots with the builtin service provider

**Comment by BataBoom** [comment]
loooking foward to this!
